nipoppy/workflow/proc_pipe/fmriprep/run_fmriprep.py

```diff
@@ -39,14 +39,14 @@ def run_fmriprep(participant_id: str,
     Path(f"{fmriprep_home_dir}").mkdir(parents=True, exist_ok=True)
 
     # BIDS DB created for fmriprep by run_nipoppy.py
-    bids_db_dir = f"/fmripre_proc/bids_db_fmriprep"
+    bids_db_dir = f"/fmriprep_proc/bids_db_fmriprep"
```

Comment:
I believe this was a typo, no?

Reply:
Yes I think so

Reply:
also fixed here in this other PR: https://github.com/neurodatascience/nipoppy/pull/178/files#diff-54dcac986f9ae4ddcf845c70c7b297b7ebef648bac1a530da650a7ca713ae2ee